src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/SemanticTokenHandler.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.LanguageServer.Common;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor.HtmlCSharp
+{
+    [Shared]
+    [ExportLspMethod(LanguageServerConstants.RazorSemanticTokensEndpoint)]
+    internal class SemanticTokenHandler : IRequestHandler<SemanticTokensParams, SemanticTokens>
```

Comment:
Do we still need this?

Reply:
+1. I suspect we shouldn't need this as the `RazorSemanticTokenEndpoint` will be called directly.